matching-notification-rest/src/main/java/org/phenotips/matchingnotification/rest/MatchingNotificationResource.java

```diff
@@ -93,6 +110,31 @@ Response getMatches(@FormParam("minScore") @DefaultValue("0.5") double minScore,
         @FormParam("fromDate") @DefaultValue("") String fromDate,
         @FormParam("toDate") @DefaultValue("") String toDate);
 
+    /**
+     * Returns a JSON object containing all matches for a provided {@code reference patient}, filtered by
+     * parameters. The following additional parameters may be specified:
+     *
+     * @param patientId the patient identifier whose matches we want to load
+     * @param minScore only matches with general score higher or equal to this value are returned
+     * @param minPhenScore only matches with phenotypic score higher or equal to this value are returned
+     * @param minGenScore only matches with genotypic score higher or equal to this value are returned
+     * @param fromDate if passed a date in the {@code yyyy-MM-dd} format, then only matches found on or after this date
+     *        will be returned; if {@code null} or an empty string, then no lower limit on the match date is considered
+     * @param toDate if passed a date in the {@code yyyy-MM-dd} format, then only matches found on or before this date
+     *        will be returned; if {@code null} or an empty string, then no upper limit on the match date is considered
+     * @return a response containing a JSON object with a list of matches
+     */
+    @GET
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/patients/{patientId}")
+    Response getMatchesForPatient(@PathParam("patientId") String patientId,
+        @FormParam("minScore") @DefaultValue("0.5") double minScore,
```

Comment:
default values should be 0.1 for average score, and 0 for gen/phen scores. There is no UI to lower the scores now, so we should load all matches except completely crappy ones (to avoid having 10s of pages of matches no on eis interested in)

Reply:
note that we wont have "pages and pages" of low-scoring matches right now, but we potentially might when we enable e.g. pubcasefinder or some other matches will return bad matches, so having at least a 0.1 limit is useful

Reply:
@marta

Reply:
@veronikaslc you got the wrong person

Reply:
@marta-